### DISCUSSION TITLE
I tried to redesign Lingmo DE's home screen.🤩

### DISCUSSION THREAD

**Discussion by Sato-Ida** [LingmoOS/Community#5] [Ideas]
Can the new design of Lingmo DE be constructed like this? I think it is very beautiful and efficient. It not only retains the aesthetics of macOS, but also has some characteristics belonging to Lingmo.😋This can also be used as a sign to distinguish Cutefish from Lingmo.🤓👆
![B4ECD8DA-B283-42AA-9650-D47B75556950](https://github.com/user-attachments/assets/1fb378c0-4b07-470a-8a11-2bd400bc52a3)
![75890C9B-37FA-4ED2-BD45-768BF2269E7A](https://github.com/user-attachments/assets/0d3cc846-ca57-4786-98ee-e5d70d893ee8)

**Comment by elysia-best** [comment]
Great!

**Comment by hanzn-zzx** [comment]
Err...It seems like deepin and harmonyOS mixed together(?
